packages/tasit-action/src/TasitAction.test.js

```diff
@@ -1,24 +1,25 @@
-import { Contract } from "./TasitAct";
+import { Contract } from "./TasitAction";
 import Account from "tasit-account";
 import { expect, assert } from "chai";
-import { waitForEvent, mineBlocks } from "./helpers/utils.js";
+import { waitForEvent, mineBlocks } from "./testHelpers/helpers";
+import { createFromPrivateKey } from "tasit-account/dist/testHelpers/helpers";
```

Comment:
Out of curiosity, do we definitely need the `dist` version?

Reply:
Babel doesn't compile `node_modules` files by default. If you want I can investigate that and figure out how to around this and use `src` file.

Reply:
Ah yes, that makes sense. We can leave it as-is for now, but we should note in a comment in the code that if the dev isn't careful this could lead to some confusing behavior while making a change that spans both files

Reply:
Is this thread can be resolved?